main.tf

```diff
@@ -0,0 +1,152 @@
+## Dynamic template for the flow logs according to the service the servcie that user want.
+ provider "aws" {
+   access_key = ""
+   secret_key = ""
```

Comment:
example usage code is missing .. along with README ..  please create README.md from chatgpt or terraform-doc